client/src/components/CheckBox/CheckBox.tsx

```diff
@@ -16,19 +16,14 @@ export interface CheckBoxProps {
   label: string; // required text for checkbox
   active?: boolean; // required, equivalent of 'checked'
   color?: string; // defaults to primaryColor
-  isSolidBox: boolean; // defaults to false
+  isSolidBox?: boolean; // defaults to false
   container_style?: React.CSSProperties; // style for div, containing checkbox and label
   checkbox_style?: React.CSSProperties; // style for checkbox
   label_style?: React.CSSProperties; // style for label
-  checkmark_vertical_align: number; // defaults to 0.1
+  checkmark_vertical_align?: number; // defaults to 0.1
```

Comment:
I chose my method because it was on the cheatsheet and it seems easy to implement. Should I remove the comments that say the default value for the props?

Reply:
Yeah, none of those comments should be necessary. Maybe give more self-descriptive prop names in some cases too, if it's too vague without the comment.